convert/src/vertex_to_bd/badgerdoc_to_vertex_use_case.py

```diff
@@ -0,0 +1,84 @@
+import tempfile
+from pathlib import Path
+from typing import NamedTuple
+
+from src.vertex_to_bd.labelstudio_format import LabelStudioFormat
+
+from .models import S3Path
+from .models.bd_annotation_model import BadgerdocAnnotation
+from .models.bd_tokens_model import Page
+
+
+class BadgerdocData(NamedTuple):
+    page: Page
+    annotation: BadgerdocAnnotation
+
+
+class BDToVertexConvertUseCase:
+    def __init__(
+        self,
+        s3_client=None,
+    ) -> None:
+        self.s3_client = s3_client
+        self.labelstudio_format = LabelStudioFormat()
+
+    def execute(
```

Comment:
May you clarify, what does this method? By the name it's not clear

Reply:
It is a common approach in naming for UseCases